src/app/nextversion/nextversion_test.go

```diff
@@ -239,18 +246,32 @@ changes:
 		{
 			name:     "Set_No_Prefix",
 			expected: "5.1.0",
+			tags:     allTags,
 			args:     "--tag-prefix chart- --output-prefix=",
 			yaml: strings.TrimSpace(`
 changes:
 - type: enhancement
   message: New feature has been added
 			`),
 		},
+		{
+			name:     "Next_When_No_Tags_Only_Deps",
+			expected: "v0.1.0",
```

Comment:
I still don't agree with this test case. I think that if there are no versions altogether on the repository, the output should _unconditionally_ be `0.0.1`

Reply:
Even if the depencency bump is a Minor?

Reply:
The thing is in this case, the value from the first version will be calculated between 0.0.1 and the actual bump, and since in this case the dependencies create a minor, that's why it's 0.1.0.
If there would be a Breaking, for example (even if there are no previous releases), the resulting version would be 1.0.0
So, do you want that, whatever is the BumpType, if there are no previous tags, the version resulting to be 0.0.1?